scripts/install_backend_python_libs.py

```diff
@@ -0,0 +1,406 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Installation script for Oppia python backend libraries."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import collections
+import os
+import shutil
+import subprocess
+
+import python_utils
+from scripts import common
+
+import pkg_resources
+
+
+def _normalize_string_that_contains_python_library_name(original_string):
+    """Python library name strings are case insensitive which means that
+    libraries are equivalent even if the casing of the library names are
+    different. This function normalizes all strings containing python library
+    names so that strings with identical libraries have identical library name
+    strings.
```

Comment:
This doc string is slightly awkward: prefer explaining what the function does & how to use it before providing context why it's needed (the first sentence should always explain _what_ the function is doing to provide quick usage context to readers).

Reply:
Done.